src/stdlib/format_timestamp.rs

```diff
@@ -1,15 +1,20 @@
 use crate::compiler::prelude::*;
 use chrono::{
     format::{strftime::StrftimeItems, Item},
-    DateTime, Utc,
+    DateTime, TimeZone, Utc,
```

Comment:
Actually, I realized we have our own `TimeZone` type that includes support for parsing `local` to indicate local time. Could we use that one instead?

https://github.com/vectordotdev/vrl/blob/04941e71487166c9746a504bf6fcddefd198b13d/src/compiler/datetime.rs#L15-L26

That should help ensure timezone handling is consistent within VRL.

Reply:
I refactored this part.

Another thing I found out is that I tried to parse the timezone during compilation, which needs to be corrected in terms of dynamically passed timezones (if the timezone is not constant but a variable).

The parsing part is moved to the function body. If my assumption is wrong, and we do not expect users to pass a variable as the timezone value, I can return the code back.